src/components/graph-vis/helpPanel.js

```diff
@@ -0,0 +1,320 @@
+        anchorOrigin={{
+          vertical: 'bottom',
+          horizontal: 'center',
+        }}
+        transformOrigin={{
+          vertical: 'top',
+          horizontal: 'center',
+        }}>
+          <Typography className={classes.typography}>
+            {children}
+          </Typography>
+      </Popover>
+    </>
+  );
+}
+
+export function HelpPanelLabel({name, label, children}) {
+  return (
+    <span>
+      {label}
+      &nbsp;
+      <HelpPanel
+        name={name}
+        children={children}
+      />
+    </span>
+  );
+}
+
+export function SearchHelp() {
+  return (
+    <HelpPanelLabel name='search-help' label='Search'>
+      <p>
+        Use this panel to search for items in the loaded dataset. Search results
+        will match the phrase used anywhere in the name of the item. For
+        example, searching for <tt>N</tt> will match Nitrogen (<tt>N</tt>), but
+        also Sodium (<tt>Na</tt>), Sodium Chloride (<tt>NaCl</tt>), and
+        Potassium Azide (<tt>KN<sub>3</sub></tt>).
+      </p>
+
+      <h5>Pinning, Neighbors, and Subspaces</h5>
+
+      <p>
+        Next to the search results, you will see icons for <b>pinning</b>, <b>neighbors</b>,
+        and <b>elemental subspace</b>.
+      </p>
+
+      <p>
+        <i>Pinning</i> an item maintains it in a list of items, so that you will
+        not lose track of it while navigating the map. The list of pinned
+        items appears as clickable badges below the search panel. You can
+        click on the <i>neighbors</i> icon in order to additionally bring in
+        the immediate neighbors of the pinned item. This provides an easy way
+        to see an item's one-hop network without having to pin all of them.
+      </p>
+
+      <p>
+        Finally, you can click on the <i>subspace</i> icon to add the item to
+        the elemental subspace. This is a special collection of items that
+        causes all materials containing any of the elements in it to appear. For
+        instance, marking <tt>N</tt> and <tt>O</tt> for inclusion in the
+        subspace will cause Nitrogen (<tt>N</tt>), Oxygen (<tt>O</tt>) to
+        appear, but also Nitrogen Dioxide (<tt>NO<sub>2</sub></tt>) and Nitrous
+        Oxide (<tt>N<sub>2</sub>O</tt>), and any other compound made exclusively
+        from nitrogen and oxygen atoms.
+      </p>
+
+      <p>
+        Displaying neighbors and adding to a subspace both imply pinning. To
+        remove either effect, you can click on the "X" on the pinned item's
+        badge. Clicking again with go on to unpin the item as well.
+      </p>
+
+      <p>
+        The pinned/neighbor/subpsace state of each material is also visible in
+        the item detail panel, which can be opened by clicking on an item, or
+        its pinned badge.
+      </p>
+
+      <p>
+        These states can also be toggled by interacting with the map directly:
+        Ctrl-click will pin an item; shift-click will pin it and show neighbors;
```

Comment:
later on you wrapped `ctrl-click` with a tag. For consistency I would wrap all buttons and actions using e.g. a `<code>Ctrl-click</code>`, also like `"Start"` will become `<code>Start</code>`

Reply:
See c616e0d8ed3811ea36a0ddc47706d450d5472463.